OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyEmailFragment.java

```diff
@@ -133,9 +127,10 @@ public void onClick(View v) {
                     addEmail();
                 }
             });
-        }
-        if (mAdditionalEmailModels.isEmpty() && mCreateKeyActivity.mAdditionalEmails != null) {
-            mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
+
+            if (mCreateKeyActivity.mAdditionalEmails != null) {
+                mEmailAdapter.addAll(mCreateKeyActivity.mAdditionalEmails);
+            }
```

Comment:
Will do new tests today for the condition but it may be not needed since i changed the code to take null cases into consideration when adding emails from the activity to the adapter.